### PR TITLE
Added danish localization

### PR DESCRIPTION
**Pull request by byrialsen**
Added danish localizatoin to the project

### CODE REVIEW

**Comment by chkr1011** [comment]
Hi, good work.
I really want to merge it but you replaced the IP addresses in the deploy script. I know that these addresses are only my personal ones. Maybe it is possible to add a different file to the repository (which is not in source control) with a list of IP addresses. Maybe some hard coded DNS names can be added like HA4IoT or Raspberry or the default name of the host when running Windows 10 IoT Core. 
Another user asked to integrate the deployment script in the build process and receiving the IP from the build configuration. What do you think? 
Best regards 
Christian

**Comment by byrialsen** [comment]
Hi.

I know that this pull request contains my personal update to the script file and ofcourse this is wrong but it is lack to I have no experience with the flow of GitHub yet. I believe I made the script commit after I created the Pull request and didn't know that it would be a part of the pull request automatically. So I closed the Pull Request again and hoped that nobody would ever see it. But you did. :-) Sorry about that. I have just added a new Pull Request to your project without the script update. Hopefully that one is better. Let me know. I still struggling with the flow of GitHub. It is quite different from centralized source control system I am used to.

**Comment by byrialsen** [comment]
Hi again. 

In respect to your question. Could it be an idea to let the script prompt you to input the IP-address manually the first time it is executed and then when entered manually the the IP-address will be written (by the script) to a seperate file not included in the main project repository. The second time you run the script the content of the file is read (contains now exactly one IP-address) and you can now choose from this IP-address or manually enter another IP-address. This way each developer/user can build up its own personalized IP-address list and each one can also add it to their own fork. 

If a user wants to maintain (delete/change) their IP-addresses they can open the file and edit it manually. But most of the time this is not needed.

Perhaps you should format the file as key/value pairs like: <ip>=<displayname>

Ex:
127.0.0.1=127.0.0.1 )

The above is autocreated but the user can edit it manually to:

127.0.0.1=localhost
or 
192.168.1.15=Raspberry PI 2 (Master)

Does it make sense. It is just a quick thought.:-)

**Comment by chkr1011** [comment]
Yes that is a possible and good solution. Do you want to implement such a solution in the Deployment Script? If you want to contribute more things I want to know what you like to do. Are you interested in Web (App etc.) or hardware or the core services of the project? We also have a community slack channel with other contributors. I want to invite you to coordinate the future releases if you agree.

Best regards
Christian

**Comment by byrialsen** [comment]
Hi.

In the first place I am trying to help out my dad making it possible for him to turn on/off his heating system in his house using an iPhone - nothing else as a start. As I myself is a 100% Microsoft guy,  helping him out with some linux based devices (arduino/Raspberry) and an iOS app doesn't interest me much :-). Thats why I initially  thought of creating a simple project myself with Windows 10 Core,, but found your project while investigated the possibilities. The project looks really nice. So I will give it a try over the next period of time even though your system can do so much more than I actually need in the first place. But if I get it to work with this particular setup to my dad, we might both go futher with some other home automation projects both for me and him. For a long time I have wanted to start some fun coding with Windows Core IoT development.

I translated it to danish to make my dad see that this was actually possible and thought I would contribute it to the project also.

If I end up using your project I might be able to help to contribute to the project later on. That could be fun:-)

To sum up quickly my most interesting skills in respect to the project are:

Microsoft.Net/C#
UWP FrameWork (have been looking into apps dev for the last year using Template10)
WPF desktop applications
Backend services like WCF etc.
UI / Graphics (mostly vectors using InkScape)

Unfortunately I haven't worked with Web application professionally so not much help there. 

/Paw

**Comment by chkr1011** [comment]
Hi,
sure. If you have any questions or problems you can ask me every time. I will try to support you. 
You can also tell your father that Alexa (Echo / Echo Dot) support is coming next release :-) Maybe this is interesting for him.

Best regards
Christian

**Comment by byrialsen** [comment]
Thanks, I might have some questions related to development. Like how do you debug controllers, website etc the easiest way. But i think I might add an issue/question later on to the project. Is'nt that the best approach? Then others might have benefit of the discussion also.

**Comment by chkr1011** [comment]
Sure. If you have questions about hardware or software as well just contact me and I will support you.
